.github/workflows/update-slow-tests.yml

```diff
@@ -6,6 +6,7 @@ on:
     - cron: "5 21 * * *"
   # Have the ability to trigger this job manually
   workflow_dispatch:
+  pull_request:
```

Comment:
This is fine, but I guess you want to gate the uploading part and only print the JSON file when it's a PR

Reply:
whoops it was just for testing